index.ts

```diff
@@ -567,6 +567,11 @@ TEST: addTests('isRepoNetworkGraph', [
 
 export const isForkedRepo = (): boolean => exists('meta[name="octolytics-dimension-repository_is_fork"][content="true"]');
 
+export const isFork = (url: URL | HTMLAnchorElement | Location = location): boolean => getRepo(url)?.path === 'fork';
+TEST: addTests('isFork', [
```

Comment:
This should be isForkingRepo. 

isFork is ambiguous. New PR, then you can release a new "patch" version afterwards